### ISSUE TITLE
Question: how could I get the prediction results? I need to create a confusion matrix

### ISSUE THREAD

**Issue by KeiraLalala** [KevinMusgrave/pytorch-metric-learning#526]
Hi there,

I tried to use the Triple Loss example in my dataset, and I need to create a confusion matrix to visualize the results. But I don't know how to get the prediction results from the get_accuracy method.  Thank you.

**Comment by KevinMusgrave** [comment]
This library doesn't yet offer prediction results (related issues #342 and #344).

You could use the [`KNeighborsClassifier` in scikit-learn.](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html)

First, compute all the embeddings for your query and reference dataset, using PyTorch. Then convert the embeddings to numpy using `embeddings.cpu().numpy()`. Pass the reference embeddings and labels to the `KNeighborsClassifier.fit` function, and then use the `predict` function on your query embeddings to get the predicted labels.
